src/components/ContactButtons.tsx

```diff
@@ -1,5 +1,5 @@
 import { type Contacts } from '@src/models/contacts';
-import { FC } from 'react';
+import { type FC } from 'react';
```

Comment:
`import type { FC } from 'react';`

I think we should import types using the above syntax so that if we are ever importing multiple types, we don't have to keep writing 'type' in front of each one. It would also make all of our code more consistent.